### DISCUSSION TITLE
Masque works on Autobar but...

### DISCUSSION THREAD

**Discussion by adorabull** [SFX-WoW/Masque#167] [Questions] [answered]
It doesn't seem to work on my ElvUI or Blizz frames for some reason. No error message, just not skinning. :(

**Comment by StormFX** [comment, marked as answer]
Re ElvUI: It's actually up to the author of the add-on to provide support for Masque, as Masque is just an engine that other authors use to skin their own project. (Masque is basically an extended library)

Re Blizz Frames: Masque has never supported the default UI. Its description specifically states,

> Masque is an add-on for World of Warcraft that provides a skinning engine **for button-based add-ons**.

Note the part in bold.

**Comment by adorabull** [comment]
Thanks, will ask ElvUI about supporting your addon! Appreciate the clarification :D
